rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -154,6 +154,13 @@ filebeat_logging_paths:
       pattern: "{{ multiline_openstack_pattern }}"
       negate: 'true'
       match: after
+  - paths:
```

Comment:
Does magnum have multiline log messages? If so, does it at least use oslo.log so that they're all uniform?